frontend/dist/css/index.css

```diff
@@ -0,0 +1,125 @@
+html,
+body {
+  color: black;
```

Comment:
feed.css has 4 space before each css property, but here we have only 1 space. Can you make this consistent?